### PR TITLE
docs: defaultChecked doesn't exist anymore

### PR DESCRIPTION
**Pull request by mthuret**
See: https://github.com/algolia/react-instantsearch/issues/52

I also add a story about it.

### CODE REVIEW

File: packages/react-instantsearch/src/widgets/Toggle.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ToggleComponent from '../components/Toggle.js';
  * @propType {string} attributeName - Name of the attribute on which to apply the `value` refinement. Required when `value` is present.
  * @propType {string} label - Label for the toggle.
  * @propType {string} value - Value of the refinement to apply on `attributeName` when checked.
- * @propType {boolean} [defaultChecked=false] - Default state of the widget. Should the toggle be checked by default?
+ * @propType {boolean} [defaultRefinement=false] - Default state of the widget. Should the toggle be checked by default?
```
Comment (Haroenv) [inline packages/react-instantsearch/src/widgets/Toggle.js]: When did the actual change of this prop happen, I think `defaultChecked` is clearer, no?

Comment (mthuret) [inline packages/react-instantsearch/src/widgets/Toggle.js]: A while ago we decided to homogenize the api of all components, meaning that if you know how to use one, you know how to user the others. So every default* where renamed to defaultRefinement :)

Comment (Haroenv) [inline packages/react-instantsearch/src/widgets/Toggle.js]: All right makes sense

**Comment by mthuret** [comment]
@Haroenv Good catch, thanks :)

**Pull request merged**